resources/lang/en_US/additional.po

```diff
@@ -135,5 +135,14 @@ msgstr "A new news is available: %1$s"
 msgid "notification.news.new.text"
 msgstr "You can watch it at %3$s"
 
+msgid "notification.angeltype.confirmed"
+msgstr "You have been confirmed as an %s angel"
```

Comment:
no suffix for the angeltype, or you will get something like "Greenroom Angel Angel" in the e-mail.